pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.es-es.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Cómo conectar un dominio de OVHcloud a un Google Site"
+excerpt: "Prepare y configure la zona DNS de su dominio OVHcloud para conectarla a un Google Site"
+updated: 2024-10-03
+---
+
+> [!primary]
+> Esta traducción ha sido generada de forma automática por nuestro partner SYSTRAN. En algunos casos puede contener términos imprecisos, como en las etiquetas de los botones o los detalles técnicos. En caso de duda, le recomendamos que consulte la versión inglesa o francesa de la guía. Si quiere ayudarnos a mejorar esta traducción, por favor, utilice el botón "Contribuir" de esta página.
+>
+
+## Objetivo
+
+Si tiene un dominio con OVHcloud y quiere conectarlo a un Google Site, Esta guía explica los pasos necesarios para preparar y configurar la zona DNS de OVHcloud con el fin de permitir la configuración de un Google Site.
+
+**Cómo conectar un dominio de OVHcloud a un Google Site**
+
+> [!warning]
+>
+> - El servicio de asistencia Google Site no tiene acceso a los parámetros de su dominio de OVHcloud y, por lo tanto, no puede aconsejarle sobre la información que deba proporcionarle.
+>
+> - La configuración, la gestión y la responsabilidad de los servicios que OVHcloud pone a su disposición recaen sobre usted. Por lo tanto, usted deberá asegurarse de que estos funcionen correctamente.<br><br> Esta guía le ayudará a realizar las operaciones más habituales. No obstante, si necesita ayuda, le recomendamos que contacte con un [proveedor especializado](/links/partner) o con el editor del servicio. Nosotros no podremos asistirle al respecto. Para más información, consulte la sección [Más](#gofurther) información de esta guía.
+>
+
+## Requisitos
+
+- Haber iniciado sesión en el [área de cliente de OVHcloud](/links/manager){.external}.
```

Comment:
We don't need the `{.external}` tags anymore, especially on links towards our own domains.